saleor/graphql/views.py

```diff
@@ -26,6 +28,22 @@
 handled_errors_logger = logging.getLogger("saleor.graphql.errors.handled")
 
 
+def tracing_wrapper(execute, sql, params, many, context):
+    with opentracing.global_tracer().start_span(operation_name="query") as span:
+        span.set_tag("component", "db")
+        span.set_tag("db.statement", sql)
+        span.set_tag("db.many", many)
```

Comment:
What is `many` and why do we need it?

Reply:
That indicates whether the ultimately invoked call is django execute() or executemany(). Thought it might become handy.

Reply:
I think it's usually obvious from the query itself (LIMIT 1 etc.).

Reply:
yep, though defining a tag would allow to make it extra search criteria on external tracing platform. Well, on second thought it is unlikely somebody gonna ever make a use of it. Will drop that in a moment.